.travis.yml

```diff
@@ -23,20 +23,23 @@ matrix:
     - stage: r10k tests
       rvm: jruby
     - stage: r10k container tests
-      dist: bionic
+      dist: focal
       language: ruby
       services:
-        # bionic uses 18.06 but need 19.03+ for buildkit so upgrade later in relevant cell
         - docker
-      rvm: 2.6.5
+      jdk: openjdk9
```

Comment:
Is this the version we want? PE ships with java 11, so I feel like we should test against that?

Reply:
It's not used in the container matrix cell at all...  I just needed something that was available on the `focal` builder, because `openjdk8` is not and will immediately cause the cell to fail. 

I tried setting it to `jdk: ` in an effort to prevent installation in that cell, but it didn't work.

Reply:
Is `openjdk11` available? I just want to minimize confusion for future us, since 9 isn't a version we've every explicitly supported. Otherwise, could you add a comment that the version isn't used / is meaningless?

Reply:
Yeah, I can try to set that globally too. I was also minimizing the blast radius of my changes to just the docker cell, but updating everything to openjdk11 is probably a reasonable solution too.

Will update PR to try it out.